src/commands/sign_transaction.js

```diff
@@ -69,6 +69,10 @@ export const actionCreator = vorpal => async ({
 		throw new ValidationError('Could not parse transaction JSON.');
 	}
 
+	if (transactionObject.error) {
+		throw new Error(transactionObject.error);
+	}
+
 	transactions.utils.verifyTransaction(transactionObject);
```

Comment:
This throws the pretty unhelpful error `Argument must be a string.` if the signature is missing. Can we be more specific?

Reply:
Maybe this is more an issue for elements?

Reply:
Probably this is an issue with elements, but also in commander we shouldn't call `verifyTransaction` too